RadixWallet/Features/CreateAccount/Coordinator/CreateAccountCoordinator+Reducer.swift

```diff
@@ -191,6 +192,9 @@ extension CreateAccountCoordinator {
 			return .none
 
 		case let .handleAccountCreated(.success(account)):
+			if state.config.shouldSendWalletCreatedEvent {
+				appEventsClient.handleEvent(.walletCreated)
+			}
```

Comment:
This event should be added when the onboarding is completed in App+Reducer, as by definition the onboarding happens when user creates a new Wallet.

Reply:
But in such case, it could be because the user restored a wallet

Reply:
Correct, then it would be better to have this code in OnboardingCoordinator when the CreateAccount completes.